src/django/Dockerfile

```diff
@@ -21,12 +19,9 @@ COPY requirements.txt /code/
 ENV CPLUS_INCLUDE_PATH=/usr/include/gdal
 ENV C_INCLUDE_PATH=/usr/include/gdal
 
-RUN echo 'alias pip=pip3' >> ~/.bashrc
```

Comment:
Same comment here as above; this isn't a bad idea _per se_, but we avoid doing this inside project containers / VMs in order to stay explicit and consistent with community standards about what's happening. Totally fine for a local development box, however.